keyserver/storage/postgres/device_keys_table.go

```diff
@@ -47,7 +47,7 @@ CREATE TABLE IF NOT EXISTS keyserver_device_keys (
 const upsertDeviceKeysSQL = "" +
 	"INSERT INTO keyserver_device_keys (user_id, device_id, ts_added_secs, key_json, stream_id, display_name)" +
 	" VALUES ($1, $2, $3, $4, $5, $6)" +
-	" ON CONFLICT ON CONSTRAINT keyserver_device_keys_unique" +
```

Comment:
@PiotrKozimor Doesn't this hurt the functionality when using postgres?

Reply:
According to [documentation](https://www.postgresql.org/docs/9.5/sql-insert.html) we can use `index_expression` or `constraint_name`. Unique constraint is index under the hood so both `keyserver_device_keys_unique` and `(user_id, device_id)` refers to the same index that is used to check for conflict.